compiler/backend/src/org/jetbrains/kotlin/codegen/inline/InlineCodegen.kt

```diff
@@ -165,37 +165,25 @@ abstract class InlineCodegen<out T : BaseExpressionCodegen>(
 
             val extension = extensionPoints[curInstr]
             if (extension != null) {
-                val start = Label()
+                var nextFreeLocalIndex = processor.nextFreeLocalIndex
+                for (local in processor.localVarsMetaInfo.currentIntervals) {
+                    val size = Type.getType(local.node.desc).size
+                    nextFreeLocalIndex = max(offsetForFinallyLocalVar + local.node.index + size, nextFreeLocalIndex)
```

Comment:
This change might fix some weird corner case with doubles/longs, but I don't actually understand why this code isn't simply using `processor.nextFreeLocalIndex`; if there is an LVT entry that was not referenced by any instruction, the LVT is broken anyway.

Reply:
Is it possible to add any test here?

Reply:
Probably not. Like I said, if we generated an LVT that actually required this loop *at all*, we'd probably break d8 compilation as well.

Reply:
Good catch!

Reply:
Look like  that  `+1` in `remapper.remap(parameters.argsSizeOnStack + 1)` allow to avoid Long/Double problems in previous not corrected variant